package RL;

message WorldState {
  message Object {
    required float pos_x = 1;
    required float pos_y = 2;
    optional float vel_x = 3;
    optional float vel_y = 4;
    optional float angle = 5;
    optional float ang_v = 6;
    optional float left_angle = 7;
    optional float right_angle = 8;
  }
  required Object robot = 1;
  required Object ball = 2;
  optional Object target = 3;
}
